packages/lib-react-components/src/Media/components/Video/Video.spec.jsx

```diff
@@ -2,35 +2,35 @@ import { render, screen } from '@testing-library/react'
 
 import Video from './Video'
 
-const video = 'https://static.zooniverse.org/fem-assets/home-video.mp4'
+const videoSrc = 'https://static.zooniverse.org/fem-assets/home-video.mp4'
```

Comment:
The Vitest env is more picky about reassigning values to a repeat variable name. In this test file, `video` was defined globally and also within the spec blocks. That's not allowed in the Vitest env.